app/scripts/migrations/121.1.test.ts

```diff
@@ -61,9 +70,169 @@ describe('migration #121.1', () => {
       },
     };
 
-    const newStorage = await migrate(oldStorage);
-    expect(newStorage.data.AccountsController).toStrictEqual(
-      mockAccountsControllerState,
-    );
+    const newStorage = await migrate(cloneDeep(oldStorage));
+
+    expect(newStorage.data).toStrictEqual(oldStorage.data);
+  });
+
+  it('does nothing if AccountsController state is missing', async () => {
```

Comment:
I added a few tests for these "expected missing state" cases, which were already handled correctly but were not tested